applications/mne_scan/plugins/eegosports/eegosports.cpp

```diff
@@ -172,7 +182,7 @@ void EEGoSports::init()
     QDate date;
     m_sOutputFilePath = settings.value(QString("EEGOSPORTS/outputFilePath"), QString("%1Sequence_01/Subject_01/%2_%3_%4_EEG_001_raw.fif").arg(m_qStringResourcePath).arg(date.currentDate().year()).arg(date.currentDate().month()).arg(date.currentDate().day())).toString();
 
-    m_sElcFilePath = settings.value(QString("EEGOSPORTS/elcFilePath"), QString("./Resources/3DLayouts/standard_waveguard64_duke.elc")).toString();
+    m_sElcFilePath = settings.value(QString("EEGOSPORTS/elcFilePath"), QString("./resources/3DLayouts/standard_waveguard64.elc")).toString();
```

Comment:
this line should be 
`settings.value(QString("EEGOSPORTS/elcFilePath"), QString(QCoreApplication::applicationDirPath() +/resources/general/3DLayouts/standard_waveguard64_duke.elc")).toString();